ocaml-lsp-server/test/e2e/__tests__/textDocument-hover.test.ts

```diff
@@ -93,8 +97,98 @@ describe("textDocument/hover", () => {
     expect(result).toMatchObject({
       contents: {
         kind: "markdown",
-        value:
-          "```ocaml\n'a -> 'a\n(** This function has a nice documentation *)\n```",
+        value: outdent`
+          \`\`\`ocaml
+          'a -> 'a
+          \`\`\`
+          ---
+          This function has a nice documentation
+          `,
+      },
+    });
+  });
+
+  it("returns type inferred under cursor with documentation with tags (markdown formatting)", async () => {
+    languageServer = await LanguageServer.startAndInitialize({
```

Comment:
Is this server being terminated somewhere?

Reply:
Hi! Thanks for the review. Isn't this snippet from the test file responsible for terminating the server?

```ts
afterEach(async () => {
    await LanguageServer.exit(languageServer);
})
```